articles/143_legacy_interface_definition.md

```diff
@@ -205,11 +220,11 @@ Depending on the type the following values are valid:
 
 - `byte`:
 
-  - an unsigned integer value in the following interval `[0, 255]`
+  - an opaque 8-bit quantity with a numerical value in the following interval `[0, 255]`
 
 - `char`:
 
-  - an integer value in the following interval `[-128, 127]`
+  - an unsigned character with a numerical value in the following interval `[0, 255]`
```

Comment:
@dirk-thomas hmm, would it make more sense to either say the character is restricted to ASCII encoding or that we're dealing with an unsigned 8-bit quantity? Character and numerical on the same sentence sound like you're implying about size and/or encoding.

Reply:
We don't prescribe any kind of semantic / encoding to these types. This only describes the data structure itself.

Reply:
Alright, then stating 8-bit quantity would make more sense (like IDL does)? IMHO saying character and not specifying encoding nor platform may be confusing.

Reply:
Imo the information "unsigned" is important here for the expectation what type that might be in each programming language (rather than signed).

Reply:
See https://github.com/ros2/design/pull/213#issuecomment-464169725 for reverting this part of the patch.